schedule/schedule-appengine/src/main/client/jasify/checkout-window/checkout-window-anonymous.html

```diff
@@ -24,7 +24,7 @@
             <div class="card">
                 <div class="card-header">
                     <h2>Create Account
-                        <small>If you don't have Jasify account, create it</small>
+                        <small>If you do not have Jasify account, create it</small>
```

Comment:
Close...
If you do not have Jasify account, create it
If you do not have Jasify a account, create it